perf/smallbank_workload/src/smallbank_tranformer.rs

```diff
@@ -28,20 +29,66 @@ use sawtooth_sdk::signing;
 
 use playlist::bytes_to_hex_str;
 use playlist::make_addresses;
-use smallbank::SmallbankTransactionPayload;
+use smallbank::{SmallbankTransactionPayload, SmallbankTransactionPayload_PayloadType};
 
 /// Transforms SmallbankTransactionPayloads into Sawtooth Transactions.
 pub struct SBPayloadTransformer<'a> {
     signer: &'a signing::Signer<'a>,
+    signature_by_customer_id: HashMap<u32, String>,
 }
 
 impl<'a> SBPayloadTransformer<'a> {
     pub fn new(signer: &'a signing::Signer) -> Self {
-        SBPayloadTransformer { signer: signer }
+        SBPayloadTransformer {
+            signer: signer,
+            signature_by_customer_id: HashMap::new(),
+        }
+    }
+
+    fn add_signature_if_create_account(
+        &mut self,
+        payload: &SmallbankTransactionPayload,
+        signature: String,
+    ) {
+        if payload.get_payload_type() == SmallbankTransactionPayload_PayloadType::CREATE_ACCOUNT {
+            self.signature_by_customer_id
+                .insert(payload.get_create_account().get_customer_id(), signature);
+        }
+    }
+
+    fn get_all_signatures(&self, customer_ids: Vec<u32>) -> Vec<String> {
```

Comment:
I think this function name is also confusing, since it isn't actually getting all signatures and it isn't clear from the name which signatures you are actually getting. Maybe something like `get_signatures_of_customers()`?